controlplane/kubeadm/controllers/controller.go

```diff
@@ -536,7 +536,12 @@ func (r *KubeadmControlPlaneReconciler) reconcileEtcdMembers(ctx context.Context
 		return ctrl.Result{}, errors.Wrap(err, "cannot get remote client to workload cluster")
 	}
 
-	removedMembers, err := workloadCluster.ReconcileEtcdMembers(ctx, nodeNames)
+	kubernetesVersion := controlPlane.KCP.Spec.Version
+	parsedVersion, err := semver.ParseTolerant(kubernetesVersion)
+	if err != nil {
+		return ctrl.Result{}, errors.Wrapf(err, "failed to parse kubernetes version %q", kubernetesVersion)
```

Comment:
do we have a convention on keep using errors.Wrapf rather than just core library with `%w`?

Reply:
I'm not sure this is documented somewhere, but we are using "github.com/pkg/errors" consistently across the code base and I'm not aware of any intention to swicth. 
Also it provides nice handling of error stacks

Reply:
>Also it provides nice handling of error stacks

I understand the use of `github.com/pkg/errors` is historical, not sure if there's real a motivation to perpetuate its usage since [golang 1.13](https://blog.golang.org/go1.13-errors). SGTM if we don't see value in making the effort to move.

Reply:
This can be discussed if think could be valuable, but I think it is something out of the scope of this PR

Reply:
sure, it's just something I keep seeing in new PRs so just wanted to get some context. Thanks.